<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>712</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Process Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/task.png</normaloff>assets/task.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(204, 204, 204);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(191, 185, 255);</string>
   </property>
   <widget class="QListWidget" name="startwindow">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>230</width>
      <height>280</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>105</red>
          <green>105</green>
          <blue>105</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="statusTip">
     <string extracomment="here"/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>Apex Legends</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Counter-Strike: Global Offensive</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Destiny</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dota</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Elden Ring</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Operating System Project</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>The Witcher 3: Wild Hunt</string>
     </property>
    </item>
   </widget>
   <widget class="QToolButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>680</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>start button</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/91821/Desktop/desk icons/Hopstarter-3d-Cartoon-Web-Coding.256.png</normaloff>C:/Users/91821/Desktop/desk icons/Hopstarter-3d-Cartoon-Web-Coding.256.png</iconset>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="popupMode">
     <enum>QToolButton::DelayedPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="closebutton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>680</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>771</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 249, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Process Manager Ready to run</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>771</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 249, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Process Counter-Strike: Global Offensive Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>771</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 249, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Process Dota Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>771</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 249, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Process Destiny Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>771</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 249, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Process Elden Ring Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>771</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 249, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Process The Witcher 3: Wild Hunt  Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>771</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 249, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Process Apex Legends Status</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>771</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 229, 255);</string>
    </property>
    <row>
     <property name="text">
      <string>1.</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2.</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3.</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4.</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5.</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6.</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7.</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>PROCESS ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>SIZE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ARRIVAL RATE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>BRUST TIME</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>COM TIME</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TR TIME</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>WAITING TIME</string>
     </property>
    </column>
   </widget>
   <widget class="QToolButton" name="tab">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>680</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(20, 20, 20);</string>
    </property>
    <property name="text">
     <string>Table</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(233, 226, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Courier New&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="run">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>680</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(26, 26, 26);</string>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QToolButton" name="close">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>680</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 31, 31);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <zorder>label7</zorder>
   <zorder>table</zorder>
   <zorder>textEdit</zorder>
   <zorder>startwindow</zorder>
   <zorder>startButton</zorder>
   <zorder>closebutton</zorder>
   <zorder>label1</zorder>
   <zorder>label2</zorder>
   <zorder>label3</zorder>
   <zorder>label4</zorder>
   <zorder>label5</zorder>
   <zorder>label6</zorder>
   <zorder>tab</zorder>
   <zorder>run</zorder>
   <zorder>close</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>startwindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>680</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closebutton</sender>
   <signal>clicked()</signal>
   <receiver>startwindow</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>680</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tab</sender>
   <signal>clicked()</signal>
   <receiver>table</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1080</x>
     <y>700</y>
    </hint>
    <hint type="destinationlabel">
     <x>767</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>clicked()</signal>
 </slots>
</ui>
